spec/features/projects/viewing_project_spec.rb

```diff
@@ -221,5 +223,73 @@
         expect(page).to have_css("h3[title='NRT AMSR2 L2B GLOBAL SWATH GSFC PROFILING ALGORITHM 2010: SURFACE PRECIPITATION, WIND SPEED OVER OCEAN, WATER VAPOR OVER OCEAN AND CLOUD LIQUID WATER OVER OCEAN V0']")
       end
     end
+
+    it 'shows details button' do
+      within '.collection-card:first-child .action-container' do
+        expect(page).to have_content('Details')
+      end
+    end
+
+    context 'when granules detail modal is opened' do
+      before :all do
+        within '.collection-card:first-child .action-container' do
+          find('a[data-target="#C14758250-LPDAAC_ECS-modal"]').click
+        end
+      end
+
+      after :all do
+        page.evaluate_script("$('#C14758250-LPDAAC_ECS-modal').modal('hide')")
+      end
+
+      it 'shows list of in-project granules' do
+        within '#C14758250-LPDAAC_ECS-modal' do
+          expect(page).to have_css('ul')
+        end
+      end
+
+      it 'shows browse imagery when available' do
+        within '#C14758250-LPDAAC_ECS-modal ul li:first-child' do
+          expect(page).to have_css('a.panel-list-thumbnail-container')
+        end
+      end
+
+      it 'shows granule title' do
+        within '#C14758250-LPDAAC_ECS-modal ul li:first-child' do
+          expect(page).to have_content('AST_L1A#00305172016231102_05182016080215.hdf')
+        end
+      end
+
+      it 'shows granule temporal' do
+        within '#C14758250-LPDAAC_ECS-modal ul li:first-child' do
+          expect(page).to have_content('Start: 2016-05-17 23:11:02')
+        end
+      end
+
+      # it 'shows a button to exclude the granule from the project' do
```

Comment:
Do we need to keep these commented out tests?  Honestly not sure -

Reply:
Yes, it was commented due to the fact that its functionality was moved to a new ticket